server/lib/icon.coffee

```diff
@@ -6,6 +6,7 @@ log = require('printit')
 module.exports = icons = {}
 market = require './market'
 
+APPS_FOLDER = '/' + path.join 'usr', 'local', 'cozy', 'apps'
```

Comment:
I understand your point but using path is a kind of good practice. It's like a guideline.